addons/im_livechat/static/src/core/web/composer_patch.js

```diff
@@ -20,6 +44,13 @@ patch(Composer.prototype, {
         }
     },
 
+    onSelectSuggestion(ev, option) {
+        super.onSelectSuggestion(ev, option);
+        if (option.label === "bot") {
+            this.props.composer.hasSubCommand = true;
```

Comment:
> I feel like this should be handled in a generic way in mail, were you actually already added part of what is needed:

I'll do that.

> Actually, is the hasSubCommand field required? Can't we deduce it from Boolean(composer.subCommand)?

The reason for using `hasSubCommand` is so that the composer and suggestion_hook will know that the current command has a subCommand aswell. So, it will fetch next suggestions. And then on user's selection it will be assigned to `composer.subCommand`.